recipes/libevent/all/test_package/conanfile.py

```diff
@@ -0,0 +1,22 @@
+from conans import ConanFile, CMake
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    def imports(self):
```

Comment:
remove imports

Reply:
I think it is needed if a tested shared library (libevent.dylib) depends on another conan-built shared library (libssl.dylib). Otherwise, the dynamic linker will fail to locate openssl.

Reply:
understood, but you can remove .so and .dll right?

Reply:
I think the situation with locating shared library dependencies is the same for all platforms.

Reply:
I see, thanks for clarifying